### PR TITLE
Move role outside this repo

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Removes ansible stuff to https://github.com/EGI-Foundation/ansible-role-fedcloud-ops

---

**Related issue :**

### CODE REVIEW

**Review by gwarf — DISMISSED** [review]
Less is more!

**Comment by enolfc** [comment]
@gwarf issues are fixed, can you take another look?

File: .github/workflows/check-links.yml [hunk 1]
```diff
@@ -20,4 +20,3 @@ jobs:
           check-modified-files-only: "yes"
           use-quiet-mode: "yes"
           use-verbose-mode: "yes"
-          base-branch: "main"
```
Comment (gwarf) [inline .github/workflows/check-links.yml:23]: Maybe better to explicitly target master, will be easier to not overlook this when/if moving to main

**Review by gwarf — APPROVED** [review]
LGTM, made a comment, please check if you feel it's relevant.

**Review by gwarf — APPROVED** [review]
LGTM.

**Pull request merged**